style/amenity-points.mss

```diff
@@ -3028,6 +3028,38 @@
       }
     }
   }
+
+  [feature = 'man_made_goods_conveyor'] {
+    [zoom >= 12] {
+      line/line-width: 1;
+      line/line-join: round;
+      line/line-cap: round;
+      line/line-color: #999999;
+
+      dash/line-width: 3.5;
+      dash/line-join: round;
+      dash/line-color: #999999;
+      dash/line-dasharray: 12,6;
```

Comment:
Rather than starting with the dash, try using half of the gap at the beginning and the end, like this: `dash/line-dasharray: 0,3,12,3;` 

Or you could split the long dash in half instead: `dash/line-dasharray: 6,6,6,0;`

Reply:
I put the second suggestion in the PR
`dash/line-dasharray: 0,3,12,3;`
![conveyor2_z18-03123](https://user-images.githubusercontent.com/9897203/78670487-1e9e8700-78de-11ea-88fc-803123a8a640.png)

`dash/line-dasharray: 6,6,6,0;`
![conveyor2_z18-6660](https://user-images.githubusercontent.com/9897203/78670509-278f5880-78de-11ea-9715-e70344af267a.png)